extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -31,14 +40,14 @@ $color-borders: #CECECE;
 
 #CreateNewWiki {
 	background: white;
-	border: 10px solid #8ac342;
+	border: 1px solid $wds-color-faint-gray;
 	color: #747474;
 	font-size: 12px;
 	font-weight: normal;
-	margin: 50px auto 0;
-	padding: 30px 30px 60px 75px;
+	margin: 4px auto 0;
+	padding: 21px 0 60px;
```

Comment:
Are those paddings and margin from Design System? Especially 21px looks strangely specific